run.py

```diff
@@ -5,4 +5,4 @@
 app = create_app(config_name)
 
 if __name__ == '__main__':
-    app.run()
+    app.run(debug=True)
```

Comment:
It is not good to set the debug here, you could use the config file for this

Reply:
Okay.